hoomd/hpmc/tune/move_size.py

```diff
@@ -310,6 +312,48 @@ class MoveSize(_InternalCustomTuner):
         rate tends towards 1. Therefore, it is recommended to pick a moderate
         maximum move size for at least the translational moves to prevent
         requiring checking periodic images.
+
+    Note:
+        In systems containing nonmonodisperse particle shapes and/or sizes, move
+        sizes for the different types should be tuned independently so that the
+        acceptances rates for the different particles are each near the target
+        acceptance ratio. Otherwise, the *global* acceptance ratio, a weighted
+        average of the acceptance ratios for each individual particle type, will
+        approach the target value, while the per-type acceptance ratios may not
+        be close to the target value. This requires creating move size tuners
+        for each type and leveraging the ``ignore_statistics`` flag of the shape
+        property of the HPMC integrator; see below for an example.
+
+    ::
+
+        mc_tuners = {}
+        for ptype in sim.state.particle_types:
+            mc_tuners[ptype] = hoomd.hpmc.tune.MoveSize.scale_solver(
+                moves=['a', 'd'],
+                target=0.2,
+                types=[ptype],
+                trigger=hoomd.trigger.Periodic(1),
+                max_translation_move=2*max_r,
+                max_rotation_move=1.0,
+            )
+            # get shape params, modify, and re-set
+            td = mc.shape[ptype]
+            td['ignore_statistics'] = True
+            mc.shape[ptype] = td
+
+        # now loop through and only acknowledge statistics of 1 type at a time
+        for acknowledged_type in sim.state.particle_types:
+            sim.operations.tuners.append(mc_tuners[acknowledged_type])
+            td = mc.shape[acknowledged_type]
+            td['ignore_statistics'] = False
+            mc.shape[acknowledged_type] = td
+            for _ in range(10):
+                sim.run(10)
+            td = mc.shape[acknowledged_type]
+            td['ignore_statistics'] = True
+            mc.shape[acknowledged_type] = td
+            sim.operations.tuners.remove(mc_tuners[acknowledged_type])
```

Comment:
Only one tune object needs to be created,  the `obj.types` attribute only needs to be change through something like `obj.types = [acknowledged_type]`.

```suggestion
        def set_statistics(mc, type, value):
            """ Get shape params, modify, and re-set."""
            tp = mc.shape[type]
            tp["ignore_statistics"] = value
            mc.shape[type] = tp

        mc_tuner = hoomd.hpmc.tune.MoveSize.scale_solver(
            moves=['a', 'd'], target=0.2, types=[ptype],
            trigger=hoomd.trigger.Periodic(1), max_translation_move=2*max_r,
            max_rotation_move=1.0,
            )
        sim.operations += mc_tuner
        
        for ptype in sim.state.particle_types:
            set_statistics(mc, ptype, True)

        # now loop through and only acknowledge statistics of 1 type at a time
        for acknowledged_type in sim.state.particle_types:
            mc_tuner.types = [acknowledged_type]
            set_statistics(mc, acknowledged_type, False)
            sim.run(100)
            set_statistics(mc, acknowledged_type, True)
```
The suggestion is also shorter, I think which should help reduce the imposing nature of it.